components/Internship/InternshipTimeline.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+function InternshipTimeline(props) {
+	const { items } = props;
+	const totalItems = items.length;
+	const numberOfActiveItems = items.filter(item => item.active).length;
+	const progressBarWidth = totalItems > 1 ? (numberOfActiveItems - 1) / (totalItems - 1) * 100 : 0;
+
+	return (
```

Comment:
Just checking, are all of these divs necessary? And if so, can we use some sort of semantic HTML, aria labelling, or something along those lines to make this more accessible?

For example, the progress bar - at the very least, let's annotate this somehow so users with screenreaders can still "see" the progress of the internship timeline.

Reply:
Bump on this.

Reply:
hm i'm actually not sure how to do this
additionally - I think it isn't necessary, since the timeline is purely decorative and not providing any valuable information.
all information in the timeline is also available right underneath it, so making it screen readable would only repeat the information twice (this is also why I decided it was unnecessary on a mobile view)

Reply:
Sure, if that's the case, then add `aria-hidden` or whatever other accessibility annotations are needed.

(the timeline does add information, namely what has "already happened". that being said, that could be inferred by a user, so not too big of a deal)